rbx_binary/src/text_deserializer.rs

```diff
@@ -401,6 +402,32 @@ impl DecodedValues {
 
                 Some(DecodedValues::Vector3(values))
             }
+            Type::Color3uint8 => {
+                let mut r = Vec::with_capacity(prop_count);
+                let mut g = Vec::with_capacity(prop_count);
+                let mut b = Vec::with_capacity(prop_count);
+
+                for _ in 0..prop_count {
+                    r.push(reader.read_u8().unwrap());
+                }
+
+                for _ in 0..prop_count {
+                    g.push(reader.read_u8().unwrap());
+                }
+
+                for _ in 0..prop_count {
+                    b.push(reader.read_u8().unwrap());
```

Comment:
here too

Reply:
`read_u8` returns a Result whereas the parent function here returns an Option. If you look at the file, all of the other read functions use `unwrap` too.